crt_portal/static/sass/custom/landing.scss

```diff
@@ -450,6 +485,12 @@ $arrow-height: 24px;
     margin-top: 0;
   }
```

Comment:
☝️ can you do me a huge favor and remove this whole `h2` rule? You didn't introduce it; it looks like it's been here for a while. But looking at the mocks vs. the current live site, this is collapsing the top margin that should be above "How to report a civil rights violation."